index.html

```diff
@@ -10,234 +10,252 @@
+            <a href="#" class="project">See Project</a>
           </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-          </div>
-          <a href="#" class="project">See Project</a>
         </div>
-      </div>
 
-      <div class="card">
-        <img class="img" src="image/card-3.svg" alt="">
-        <div class="content">
-          <div class="topic">Tonic</div>
-          <div class="sub-topic">
-            <span>CANOPY</span>
-            <span class="dot"></span>
+        <div class="card">
+          <div class="content">
+            <div class="topic">
+              Multi-Post
+              <br />
+              Stories
+            </div>
+            <div class="sub-topic">
+              <span>CANOPY</span>
+              <span class="dot"></span>
 
-            <span>Back End Dev</span>
-            <span class="dot"></span>
+              <span>Back End Dev</span>
+              <span class="dot"></span>
 
-            <span>2015</span>
+              <span>2015</span>
+            </div>
+            <div class="description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </div>
+            <div class="button-links">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <a href="#" class="project">See Project</a>
           </div>
-          <div class="description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
-          </div>
-          <a href="#" class="project">See Project</a>
+          <img class="img" src="image/card-4.svg" alt="" />
         </div>
       </div>
+    </div>
 
-      <div class="card">
-        <img class="img" src="image/card-4.svg" alt="">
-        <div class="content">
-          <div class="topic">Multi-Post <br /> Stories</div>
-          <div class="sub-topic">
-            <span>CANOPY</span>
-            <span class="dot"></span>
-
-            <span>Back End Dev</span>
-            <span class="dot"></span>
-
-            <span>2015</span>
-          </div>
-          <div class="description">
-            A daily selection of privately personalized reads;
-            no accounts or sign-ups required.
-          </div>
-          <div class="button-links">
-            <ul>
-              <li>html</li>
-              <li>css</li>
-              <li>javascript</li>
-            </ul>
+    <div class="myself">
+      <div class="left">
+        <div class="description">
+          <h1 class="topic">About Myself</h1>
```

Comment:
- [ ] I think here you can use `h2` or `h3` instead of `h1`  because you should use it first to follow the order of the heading rules for accessibility issues💯 and also make sure that you use it one per page because Having a unique H1 tag would be helpful in regards to Google recognizing unique relevance between different pages.

- [ ] Kindly make the font size the same as the Figma 💯💯